lib/src/hash_sink.dart

```diff
@@ -44,6 +44,9 @@ abstract class HashSink implements Sink<List<int>> {
   /// This should be updated each time [updateHash] is called.
   Uint32List get digest;
 
+  /// The length
+  int get signatureBytes => 8;
```

Comment:
Alternative view: Is this really something the *user* of this class needs to see? If it's only used internally in the private members of this base class, then it does not need to be part of the public API.

Could it instead be a private fields initialized by the constructor, so subclasses provide the value in their constructor `super` invocation?
You can even validate the value in the constructor then.

Unless there is a good argument for making the value part of the public API, I think I'd prefer the other approach.

Reply:
That sounds perfectly reasonable to me. Note: hashsink is not a public API. lib/crypto.dart does not export it.